pkg/discovery/k8s/k8s.go

```diff
@@ -0,0 +1,106 @@
+package k8s
+
+import (
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap.com/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/client/clientset/versioned"
+	"github.com/pingcap/tidb-operator/pkg/discovery"
+	"github.com/pingcap/tidb-operator/pkg/discovery/server"
+	"github.com/pingcap/tidb-operator/pkg/pdapi"
+	"github.com/pingcap/tidb-operator/pkg/version"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/util/logs"
+	"k8s.io/client-go/rest"
+)
+
+// PrintVerstionInfo calls version.PrintVersionInfo
```

Comment:
Why wraps these two functions? The `version` package is already pretty small and standalone.